flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/stats/CatalogColumnStatisticsDataLong.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.catalog.stats;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Column statistics value of long type.
+ */
+public class CatalogColumnStatisticsDataLong extends CatalogColumnStatisticsDataBase {
+	/**
+	 * mim value.
+	 */
+	private final long min;
+
+	/**
+	 * max value.
+	 */
+	private final long max;
+
+	/**
+	 * number of distinct values.
+	 */
+	private final long ndv;
```

Comment:
move ndv to `CatalogColumnStatisticsDataBase`? I think it's valid for all types of columns.

Reply:
Maybe true, but Hive doesn't give ndv for boolean. I'm pretty much following Hive's stats definition. Let me know if you insist putting ndv for all types.

Reply:
I'm okay for this, and i just realized that it's also weird for `double` and `float` have ndv

Reply:
Yeah. Hive uses double stats for both "double" and "float". Theoretically, ndvs for double might not be accurate, but from statistical point view, the value is still there. It's just that it cannot be used to answer queries such as select count(distinct).